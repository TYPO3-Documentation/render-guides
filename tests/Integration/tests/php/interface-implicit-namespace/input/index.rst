=====================================
PHP Interface with implicit namespace
=====================================

..  php:interface:: TYPO3\CMS\Core\TestInterface

    Lorem Ipsum Dolor!
