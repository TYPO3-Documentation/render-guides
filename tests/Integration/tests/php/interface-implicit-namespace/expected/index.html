<!-- content start -->
    <div class="section" id="php-interface-with-implicit-namespace">
    <h1>PHP Interface with implicit namespace</h1>

            <dl class="php interface">
    <dt class="sig sig-object php"
        id="typo3-cms-core-testinterface">
        <em class="property"><span class="pre">interface</span> </em>
            <span class="sig-prename descclassname"><span class="pre">\TYPO3\CMS\Core\</span></span>
<span class="sig-name descname"><span class="pre">TestInterface</span></span>

    </dt>
    <dd>
        <p>Lorem Ipsum Dolor!</p>
    </dd>
</dl>

    </div>

<!-- content end -->